README.md

```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
 
 <!--RULES_TABLE_END-->
 
+## :couple: FAQ
+
+### What is "Use the latest vue-eslint-parser" errors?
+
+The most rules of `eslint-plugin-vue` requires `vue-eslint-parser` to check `<template>` ASTs.
+
+Please add one of the following settings into your configuration:
+
+- `"extends": ["plugin:vue/recommended"]`
+- `"extends": ["plugin:vue/base"]`
+- `"parser": "vue-eslint-parser"` (and do `npm install -D vue-eslint-parser`)
```

Comment:
I think we can remove the third option and force using base or recommended configuration. The less options the better, people will not get confused :)